### PR TITLE
PN-133

### PR DESCRIPTION
**Pull request by itaiGershtansky**
Added methods.

### CODE REVIEW

**Comment by itaiGershtansky** [comment]
@buske, please review

**Comment by buske** [comment]
+1

File: similarity-data-api/src/main/java/org/phenotips/data/similarity/PatientSimilarityView.java [hunk 1]
```diff
@@ -70,4 +70,20 @@
      *         match, with {@code 0} for patients with no similarities
      */
     double getScore();
+
+    /**
+     * Get the phenotypic similarity score for this patient match.
+     *
+     * @return the similarity score, between 0 (a poor match) and 1 (a good
```
Comment (sdumitriu) [inline similarity-data-api/src/main/java/org/phenotips/data/similarity/PatientSimilarityView.java]: @buske I see that the Javadoc for `getScore()` (see above) uses `[-1, 1]` for the score range. Is that valid or deprecated?

Comment (buske) [inline similarity-data-api/src/main/java/org/phenotips/data/similarity/PatientSimilarityView.java]: Scores are never negative, so we could consider it deprecated. I think the concept of an "opposite" patient description is rather tenuous.

**Pull request merged**